### DISCUSSION TITLE
What is the definition of `gt_classes`?

### DISCUSSION THREAD

**Discussion by keiohta** [3dlg-hcvc/OPD#2] [Q&A] [answered]
Hi authors,

Thanks for sharing the great codebase. I am trying to train OPD from scratch on my own dataset, but I'm not sure what value should I put to the `gt_classes`. I thought it is related to articulation type but you've already used it for `gt_types` (drawer, door, or lid). So, could you tell me what information should I put to the `gt_classes`?

Thanks,
Kei

**Comment by Jianghanxiao** [comment, marked as answer]
Thanks for your questions!! In gt_classes, it contains the part categories. Because we want to detect different parts. For the motion type, it's the motion attribute of the part. We don't encode it into the gt_classes. If you need more help, feel free to post here.

**Reply by keiohta** [reply]
Hi @Jianghanxiao, thanks for the answer! and sorry for the late reply. Let me ask two more questions to clarify my understanding.

> In gt_classes, it contains the part categories. Because we want to detect different parts.

Is it correct that `part categories` consists of {drawer, door, lid}?

> For the motion type, it's the motion attribute of the part.

Also, the `motion type` consists of {prismatic, revolute}?

**Reply by Jianghanxiao** [reply]
Yes, it's correct that part categories consists of {drawer, door, lid}. And it's correct that the motion type consists of {prismatic, revolute}

**Reply by keiohta** [reply]
Thank you for the answer. Now I can train OPD from scratch on our dataset. Thanks again!
